src/components/specific/models/models-manager/generic-models-manager/GenericModelsManager.scss

```diff
@@ -21,6 +21,34 @@
     color: var(--color-white);
   }
 
+  &__building-maker-btn {
```

Comment:
Are these styles still necessary ?
It looks like it is a duplicate of those you added in `PDFManager.scss`.